### PR TITLE
Update to the latest 2.9 schema

### PR DESCRIPTION
**Pull request by SimonRichardson**
The following updates the 2.9 branch to the latest schema. The generated
python is the fallout from this.

### CODE REVIEW

File: juju/client/_client11.py [hunk 1]
```diff
@@ -3697,27 +3692,6 @@ async def SetSupportedContainers(self, params=None):
 
 
 
-    @ReturnMapping(StringsResult)
-    async def StateAddresses(self):
```
Comment (SimonRichardson) [inline juju/client/_client11.py:3701]: This is a bit scary...

Comment (achilleasa) [reply juju/client/_client11.py:3701]: Why is that?

**Review by achilleasa — APPROVED** [review]

File: juju/client/_client11.py (hunk 1 above)
Comment (SimonRichardson) [reply juju/client/_client11.py:3701]: As we shouldn't be removing methods from the API server i.e. it's always additive. 

I believe this is ok, as it's a change @wallyworld did https://github.com/juju/juju/pull/12184

**Comment by SimonRichardson** [comment]
$$merge$$

**Pull request merged**